ml_model/src/test/java/edu/wm/sealab/featureextraction/FeatureExtractorTest.java

```diff
@@ -31,29 +36,28 @@ public void setup() {
     File file = new File(filePath);
 
     CompilationUnit cu = null;
+    CompilationUnit cuNoComm = null;
     try {
       cu = StaticJavaParser.parse(file);
+      JavaParser parser = new JavaParser(new ParserConfiguration().setAttributeComments(false));
+      cuNoComm = parser.parse(file).getResult().get();
     } catch (FileNotFoundException e) {
       e.printStackTrace();
     }
 
     featureVisitor = new FeatureVisitor();
 
-    syntacticFeatureExtractor = new SyntacticFeatureExtractor(featureVisitor.getFeatures());
-
-    String codeSnippet = null;
-    try {
-      codeSnippet = Files.readAllLines(path).toString();
-    } catch (IOException e) {
-      e.printStackTrace();
-    }
-
     // Capture Java Parser related features eg: #ifstmts
     featureVisitor.visit(cu, null);
 
-    // Captures non-Java Parser related features eg: #parenthesis
-    syntacticFeatureExtractor.extract(codeSnippet);
-  }
+    // Modify the CU to compute syntactic features i.e. parenthesis, commas, etc
+    StringLiteralReplacer stringLiteralReplacer = new StringLiteralReplacer();
+    stringLiteralReplacer.visit(cuNoComm, null);
```

Comment:
Why do we need this 2 lines of code?

Reply:
I was mirroring the processes from Parser.java to ensure that the parenthesis inside comments would not be counted. It looks like those lines modify cuNoComm, but I could be wrong. Should I remove them?